### PR TITLE
Remove Config-ID from examples and fix Audit tools

### PR DESCRIPTION
**Pull request by dbenders**

### CODE REVIEW

File: packages/pangea-sdk/README.md [hunk 1]
```diff
@@ -130,7 +126,7 @@ It accepts multiple file formats:
 - a search response from the REST API:
 
 ```
-curl -H "Authorization: Bearer ${PANGEA_TOKEN}" -X POST -H 'Content-Type: application/json'  --data '{"verbose": true}' https://audit.dev.aws.pangea.cloud/v1/search
+curl -H "Authorization: Bearer ${PANGEA_TOKEN}" -X POST -H 'Content-Type: application/json'  --data '{"verbose": true}' https://audit.pangea.cloud/v1/search
```
Comment (govind-salinas-pangea) [inline packages/pangea-sdk/README.md:129]: this should be audit.aws.us.pangea.cloud

File: packages/pangea-sdk/examples/demo_app/README.md [hunk 2]
```diff
@@ -10,10 +10,7 @@ The App simulates an HR application that allows a user to upload resumes, retrie
 ### Environment Setup
 Set the following environment variables:
 - `PANGEA_TOKEN`
-- `EMBARGO_CONFIG_ID`
-- `REDACT_CONFIG_ID`
-- `AUDIT_CONFIG_ID`
-- `PANGEA_DOMAIN` (ex: "aws.us.pangea.cloud")
+- `PANGEA_DOMAIN` (ex: "pangea.cloud")
```
Comment (govind-salinas-pangea) [inline packages/pangea-sdk/examples/demo_app/README.md:13]: this was correct before

File: packages/pangea-sdk/examples/demo_app/README.md [hunk 3]
```diff
@@ -99,4 +96,4 @@ sqlite3 demo-app.db
 ```
 
 ## References
-Viewing Audit Logs: https://console.dev.pangea.cloud/service/audit/logs
+Viewing Audit Logs: https://console.pangea.cloud/service/audit/logs
```
Comment (govind-salinas-pangea) [inline packages/pangea-sdk/examples/demo_app/README.md:99]: fix domain

File: packages/pangea-sdk/pangea/audit_logger.py [hunk 4]
```diff
@@ -79,7 +79,7 @@ def initLogging(domain: str, token: str):
     """Initializes Audit logging environment
 
     Args:
-        domain (string) : the Pangea domain to use, i.e. "aws.us.pangea.cloud"
+        domain (string) : the Pangea domain to use, i.e. "pangea.cloud"
```
Comment (govind-salinas-pangea) [inline packages/pangea-sdk/pangea/audit_logger.py:82]: fix domain

File: packages/pangea-sdk/pangea/services/embargo.py [hunk 5]
```diff
@@ -56,7 +56,7 @@ class Embargo(ServiceBase):
     """Embargo service client.
 
     Provides methods to interact with Pangea Embargo Service:
-        https://docs.dev.pangea.cloud/docs/api/embargo
+        https://pangea.cloud/docs/api/embargo
```
Comment (govind-salinas-pangea) [inline packages/pangea-sdk/pangea/services/embargo.py:59]: should be docs.pangea.cloud (no aws.us here)

File: packages/pangea-sdk/pangea/services/redact.py [hunk 6]
```diff
@@ -107,7 +107,7 @@ class Redact(ServiceBase):
     """Redact service client.
 
     Provides the methods to interact with the Pangea Redact Service:
-        [https://docs.dev.pangea.cloud/docs/api/redact](https://docs.dev.pangea.cloud/docs/api/redact)
+        [https://pangea.cloud/docs/api/redact](https://pangea.cloud/docs/api/redact)
```
Comment (govind-salinas-pangea) [inline packages/pangea-sdk/pangea/services/redact.py:110]: need to keep the service name in the domain

File: packages/pangea-sdk/pangea/services/embargo.py (hunk 5 above)
Comment (dbenders) [reply packages/pangea-sdk/pangea/services/embargo.py:59]: According to @r3dcrosse it is https://pangea.cloud/docs/xxx

File: packages/pangea-sdk/pangea/services/redact.py (hunk 6 above)
Comment (dbenders) [reply packages/pangea-sdk/pangea/services/redact.py:110]: according to @r3dcrosse it's ok

**Review by govind-salinas-pangea — APPROVED** [review]

**Pull request merged**